docs/design/admin_control_over_spec.md

```diff
@@ -0,0 +1,69 @@
+# Design proposal template `Admin users control over non admin objects' spec`
+
+## Abstract
+
+Non Admin Controller (NAC) restricts the usage of OADP operator with NonAdminBackups and NonAdminRestores.
+Admin users may want to further restrict this by restricting NonAdminBackup/NonAdminRestore spec fields values.
+
+## Background
+
+Non Admin Controller (NAC) adds the ability to admin users restrict the use of OADP operator for non admin users, by only allowing them to create backup/restores from their namespaces with NonAdminBackups/NonAdminRestores.
+Admin users may want to further restrict non admin users operations, like forcing a specific time to live (TTL) for NonAdminBackups associated Velero Backups.
+This design enables admin users to set custom default values for NonAdminBackup/NonAdminRestore spec fields, which can not be overridden.
+
+## Goals
+
+Enable admin users to
+- set custom default values for NonAdminBackup spec.backupSpec fields, which can not be overridden
+- TODO restore
+
+## Non Goals
+
+- Show NonAdminBackup spec.backupSpec fields/TODO NonAdminRestore custom default values to non admin users
+- Do not allow non admin users to create NonAdminBackup/NonAdminRestore with overridden defaults (but they will not validated by NAC, with an error explaining why in the object statuses and in NAC logs)
+- Allow admin users to set second level defaults (for example, NonAdminBackup `spec.backupSpec.labelSelector` can have a custom default value, but not just `spec.backupSpec.labelSelector.matchLabels`)
+
+
+## High-Level Design
+
+A field will be added to OADP DPA object. With it, admin users will be able to select which NonAdminBackup `spec.backupSpec` fields have custom default (and enforced) values. NAC will respect the set values. If a NonAdminBackup is created with fields overriding any enforced values, it will fail validation prior to creating an associated Velero Backup. If admin user changes any enforced field value, NAC Pod is recreated to always be up to date with admin user enforcements.
+
+TODO restore
+
+
+## Detailed Design
+
+Field `spec.nonAdmin.enforceBackupSpec`, of the same type as the Velero Backup Spec, will be added to OADP DPA object.
```

Comment:
Please give an example of DPA yaml file here. May go over a use-case/scenario of how the workflow would be ?
- Sample DPA configured by admin
- Sample NAB case accepting value from Non-Admin user and al respecting some defaults from Admin enforced values